src/system.js

```diff
@@ -92,6 +93,7 @@ export {
   Selector,
   SiteSearch,
   Slider,
+  SliderTwo,
```

Comment:
We should name it something else. This can be changed in the final release but don't think `SliderTwo` is distinctive enough.

Reply:
I agree, this is just for testing so it doesnt mess up anything else